frontend/views/containers/payments/MonthOverview.vue

```diff
@@ -2,9 +2,9 @@
 .c-summary(data-test='monthOverview')
   i18n.c-summary-title.is-title-4(
     tag='h4'
-    data-test='thisMonth'
-    :args='{ month: humanDate(Date.now(), { month: "long" }) }'
-  ) {month} overview
+    data-test='monthOverviewTitle'
+    :args='{ start: humanDate(startDate), end: humanDate(dueDate) }'
```

Comment:
Instead of calling `humanDate` here, it's better to call it inside of the computed properties `startDate` and `dueDate` where it's easier to customize it while keeping the markup clean and free of JS as much as possible.